libs/services/intl/src/compiled-lang/strings.en-XA.json

```diff
@@ -11727,6 +11727,20 @@
       "value": "]"
     }
   ],
+  "W7kvPF": [
+    {
+      "type": 0,
+      "value": "["
+    },
+    {
+      "type": 0,
+      "value": "Ŧħḗḗ ḗḗẋƥřḗḗşşīǿǿƞ īş īƞṽȧȧŀīḓ. Ƥŀḗḗȧȧşḗḗ ŭŭşḗḗ şīƞɠŀḗḗ ɋŭŭǿǿŧḗḗş."
```

Comment:
Not sure how to fix this? Use wording as previously suggested.

Reply:
done! this file gets automatically updated based on the other ones